### ISSUE TITLE
Import material icons from library

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#28580]
Since we decided using [Material icons](https://material.io/tools/icons/) on our Gutenberg blocks (https://github.com/Automattic/wp-calypso/issues/28519, https://github.com/Automattic/wp-calypso/issues/28500), we'll have plenty more of these coming up.

Would be great to be able to import these directly from a library instead of manually copying them over like we're doing now.

We'll need the actual svg contents, not just image path and image copied with bundle like we do for the rest of the images.

Some candidate source libraries:
- https://www.npmjs.com/package/@material-ui/icons
- https://www.npmjs.com/package/material-design-icons

(via https://github.com/Automattic/wp-calypso/pull/28501#issuecomment-438417992)

cc @Automattic/luna @Automattic/poseidon

**Comment by blowery** [comment]
Ah, looks like Gutenberg has just been inlining them as needed. 

It looks like Gutenberg has a custom SVG wrapper for a11y concerns. Rather than pulling into one of those third party components, which has it's own set of opinions, would it make sense to add a new icons package to Gutenberg and expose the icons that way? 

@material-ui/icons returns a ReactElement with their custom wrapper
material-design-icons hasn't been updated in ... 2 years? /cc @lynnmercier

**Comment by simison** [comment]
>It looks like Gutenberg has a custom SVG wrapper for a11y concerns.

Yep! [Gutenberg SVG primitives](https://github.com/WordPress/gutenberg/tree/ec1a4cbe126d860f0feb0dda8c0c8f8c1d38419b/packages/components/src/primitives/svg#readme) add these three attributes for better accessibility:

    role: 'img',
    'aria-hidden': 'true',
    focusable: 'false',

We should try to PR these to upstream to a library we choose to use so that it benefits everyone, not just us.

**Comment by simison** [comment]
Noting that Gutenberg's SVG wrapper is not only for a11y concerns but also to make SVGs compatible with React Native:

https://github.com/WordPress/gutenberg/blob/ec1a4cbe126d860f0feb0dda8c0c8f8c1d38419b/packages/components/src/primitives/svg/index.native.js

**Comment by simison** [comment]
Convo is up again https://make.wordpress.org/design/2019/02/06/meeting-notes-for-february-6-2018/
